python/requirements.txt

```diff
@@ -7,6 +7,7 @@ nest_asyncio
 networkx
 numpy
 pandas
+progressbar
```

Comment:
Will tqdm be simpler?

Using `progressbar` is also OK to me.

Reply:
I agree to use `tqdm`